src/swagger-specs/query-service.yaml

```diff
@@ -1278,38 +1601,72 @@ definitions:
           method:
             type: string
             description: the HTTP method to use
+  
+  PagingProperties:
+    type: object
+    description: Contains properties to describe the order, size, total number of pages, and current page.
+    properties:
+      orderby:
+        type: string
+        description: The property by which this page is ordered.
+        example: '-created'
+      page:
+        type: integer
+        description: A page of results.
```

Comment:
From Chhavi:
description: A page of results.
can be changed to
Page number of the result.